### PR TITLE
doc: doc team structure and related projects

### PR DESCRIPTION
**Pull request by joyeecheung**
Fixes: https://github.com/nodejs/automation/issues/8
Refs: https://github.com/nodejs/automation/issues/1 (related projects)
Refs: https://github.com/nodejs/automation/issues/12 (monorepo proposal)
 
### Team structure

This is based on the idea proposed in https://github.com/nodejs/automation/issues/8#issuecomment-341423820. I have created the subteams described in the PR.

- The initial members in the two subteams @nodejs/automation-collaborators @nodejs/automation-admins are people in TSC and already in @nodejs/automation, because TSC are owners of the organization so they have write/admin access to these repos already.
- In addition I have added @rvagg who is both TSC and previous owners of commit stream, changelog-maker and branch-diff to both teams.
- If you want to join the subteams to get write/admin access, <del>please let me know on gitter or send me an email to the address in my profile.</del> I am not supposed to make the decision alone, will defer to another thread to discuss about this.

### Relate projects

- The documented related projects are automation-related projects that have already been transferred into the foundation. 
- To discuss about moving more outside projects, please reply in https://github.com/nodejs/automation/issues/1 or open a separate issue.
- If you do not think the documented projects should be owned by the automation team, please discuss here.
- To discuss about monorepo, please reply in https://github.com/nodejs/automation/issues/12. 
Until we come to a conclusion there, we will need to document the related projects separately.

The question of npm access seems to be less urgent, we can just ask the existing releasers to do the work and revisit later when there are not enough of them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,37 @@
...
-## Projects
+The automation team is currently structured like this:
 
-To be discussed.
+- **@nodejs/automation**: people to ping for automation-related issues in
+the organization.
+  - **@nodejs/automation-collaborators**: has write access to all
+  automation-related repositories.
+    - **@nodejs/automation-admins**: has admin access to all
+    automatino-related repositories.
```
Comment (richardlau) [inline README.md]: Typo here

Comment (joyeecheung) [reply README.md]: Thanks for catching that, fixed

**Review by Tiriel — APPROVED** [review]

**Review by priyank-p — APPROVED** [review]

**Review by alopezsanchez — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -4,12 +4,37 @@
...
+  - Turn a `git log` into a stream of commit objects.
+- [core-validate-commit](https://github.com/nodejs/core-validate-commit)
+  - Validate commit messages for Node.js core.
+- [github-bot](https://github.com/nodejs/github-bot)
+  - @nodejs-github-bot's heart and soul.
+- [node-auto-test](https://github.com/nodejs/node-auto-test)
+  - A [nodejs/node](https://github.com/nodejs/node) clone for testing tools
+    and bots.
+- [node-review](https://github.com/nodejs/node-review)
+  - Generate the metadata required for landing a Node.js Pull Request.
```
Comment (evanlucas) [inline README.md]: Maybe be more specific here that this is a browser extension?

Comment (joyeecheung) [reply README.md]: @evanlucas I copied it from the repo description, will update that one as well, thanks!

```
A browser extension that generates the metadata required for landing a Node.js Pull Request
```

**Review by evanlucas — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -4,12 +4,38 @@
...
 
-To be discussed.
+- **@nodejs/automation**: people to ping for automation-related issues in
+the organization.
+  - **@nodejs/automation-collaborators**: has write access to all
+  automation-related repositories.
+    - **@nodejs/automation-admins**: has admin access to all
+    automation-related repositories.
 
+## Projects / Repositories
```
Comment (maclover7) [inline README.md:21]: needs `lts` tool as well

**Review by maclover7 — COMMENTED** [review]
+1 once `lts` is added

File: README.md (hunk 3 above)
Comment (joyeecheung) [reply README.md:21]: Do you mean https://github.com/continuationlabs/lts ? It has not been transferred into the foundation yet..

Comment (joyeecheung) [reply README.md:21]: I guess we will wait for https://github.com/nodejs/Release/pull/277 and for it to be transferred (need an issue in the TSC repo first?), we probably should not delay this PR for that, can add it later once it's been transferred.

Comment (maclover7) [reply README.md:21]: Yep that's the one -- I thought it had already been transferred to the nodejs org... cc @cjihrig

**Review by riyadhalnur — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (gibfahn) [reply README.md:21]: https://github.com/nodejs/TSC/issues/410

**Review by aashil — APPROVED** [review]

**Comment by joyeecheung** [comment]
@maclover7 I suggest we add lts later once it's transferred. For now I will just merge this so our README gives more information on the team.

**Pull request merged**